emperor/support_files/js/sceneplotview3d.js

```diff
@@ -685,10 +685,13 @@ define([
       this.control.update();
     }
 
-    //point all samples towards the camera
-    _.each(this.decViews.scatter.markers, function(element) {
-      element.quaternion.copy(camera.quaternion);
-    });
+    // don't point arrows towards the camera, that makes zero sense :L!
+    if (this.decViews.scatter.decomp.isScatterType()) {
+      //point all samples towards the camera
```

Comment:
this comment seems to contradict the prior...

Reply:
Good catch, thanks. I've rephrased these comments for them to make sense.